lib/optimizely/helpers/constants.rb

```diff
@@ -327,6 +327,8 @@ module Constants
         'v3' => '3',
         'v4' => '4'
       }.freeze
+
+      ATTRIBUTE_VALID_TYPES = [FalseClass, Fixnum, Float, TrueClass, String].freeze
```

Comment:
What about Integer?
And we need to check strictly False primitive type. 
We don't want to allow any class inheriting from False or True classes. Please make sure